osmoutils/accum/accum_test.go

```diff
@@ -1711,3 +1711,82 @@ func (suite *AccumTestSuite) TestAddToUnclaimedRewards() {
 		})
 	}
 }
+
+func (suite *AccumTestSuite) TestDeletePosition() {
+	// We setup store and accum
+	// once at beginning.
+	suite.SetupTest()
```

Comment:
You should either run setup here or below in the for loop but not both

Reply:
Copied over and didn't notice, removed now